lib/netsuite/records/markup_item.rb

```diff
@@ -19,12 +19,11 @@ class MarkupItem
 
       record_refs :klass,
                   :department,
-                  :expense_account,
+                  :account,
                   :location,
                   :tax_schedule
 
       field :custom_field_list, CustomFieldList
```

Comment:
does item vendor list not exist?

Reply:
oops think it was my bad!

Reply:
I didn't see it in NS [here](https://7232422-sb1.app.netsuite.com/app/common/item/item.nl?id=16&e=T)

Reply:
Only vendor name which I don't think is really the same, its freeform text vs a vendor ref

Reply:
ya sorry - i meant that i checked NS as well and realized i messed up earlier!

Reply:
Oh gotcha! no worries at all